tensorflow/python/kernel_tests/signal/fft_ops_test.py

```diff
@@ -554,6 +554,47 @@ def testGrad_Random(self):
             self._checkGradComplex(
                 self._tfIFFTForRank(rank), re, im, result_is_complex=False)
 
+class FFTShiftTest(test.TestCase):
+
+  def testDefinition(self):
+    x = [0, 1, 2, 3, 4, -4, -3, -2, -1]
+    y = [-4, -3, -2, -1, 0, 1, 2, 3, 4]
+    self.assertTrue((fft_ops.fftshift(x).numpy() ==  y).all())
+    self.assertTrue((fft_ops.ifftshift(y).numpy() ==  x).all())
+    x = [0, 1, 2, 3, 4, -5, -4, -3, -2, -1]
+    y = [-5, -4, -3, -2, -1, 0, 1, 2, 3, 4]
+    self.assertTrue((fft_ops.fftshift(x).numpy() == y).all())
+    self.assertTrue((fft_ops.ifftshift(y).numpy() ==  x).all())
+
+  def testAxesKeyword(self):
+    freqs = [[0, 1, 2], [3, 4, -4], [-3, -2, -1]]
+    shifted = [[-1, -3, -2], [2, 0, 1], [-4, 3, 4]]
+    self.assertTrue((fft_ops.fftshift(freqs, axes=(0, 1)).numpy() == \
```

Comment:
note, the style guide disallows the use of \ as a line continuation
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#32-line-length